src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -40,4 +40,29 @@ package object stage {
       }
     }
   }.isSuccess
+
+
+  /*
+   Logic opperators that work with Try[Boolean]
+   */
+  implicit class TryLogic(val t: Try[Boolean]) extends AnyVal {
```

Comment:
At the time I didn't have both versions available and wanted a symmetrical API so I left out some parts.